graylog2-server/src/main/java/org/graylog2/configuration/ElasticsearchConfiguration.java

```diff
@@ -96,6 +99,16 @@ public class ElasticsearchConfiguration {
     @Parameter(value = "elasticsearch_max_number_of_indices", required = true, validator = PositiveIntegerValidator.class)
     private int maxNumberOfIndices = 20;
 
+    // TimeBasedSizeOptimizingStrategy Rotation
+    @Parameter(value = "time_size_optimizing_rotation_period")
+    private Period timeSizeOptimizingRotationPeriod = Period.days(1);
```

Comment:
Do we need to make sure rotation period is a multiple of full days, or an integer division of a day? Can anything bad happen if this is changed to e.g. 7 hours?

Reply:
No, I'm currently testing it with minutes `PT1M`